doc/conf.py

```diff
@@ -17,6 +17,9 @@
 import matplotlib
 import sphinx
 
+if sys.version_info <= (3, 0, 0):
+    print("You're using python 2.x, conf.py works with python3+ only.")
```

Comment:
This should fail or otherwise exit here, or any Python 3 code later that fails to parse will cause a confusing second error.